### PR TITLE
Update `support/xgettext.pl`

### PR DESCRIPTION
**Pull request by ikedas**
Update `support/xgettext.pl`
  - `cpanfile` is no longer scanned.
  - add `#, smalltalk-format` flag to maketext'ed messages.
  - make some fields in the header updated.
  - suppress informational outputs by default.

### CODE REVIEW

**Review by S1aa — APPROVED** [review]

File: support/xgettext.pl [hunk 1]
```diff
@@ -6,8 +6,9 @@
 use warnings;
 use Cwd qw();
 use English;    # FIXME: Avoid $MATCH usage
-use Getopt::Long;
```
Comment (S1aa) [inline support/xgettext.pl:9]: TOP.123

File: po/sympa/Makevars [hunk 2]
```diff
@@ -43,7 +43,7 @@ PACKAGE_GNU = no
 # It can be your email address, or a mailing list address where translators
 # can write to without being subscribed, or the URL of a web page through
 # which the translators can contact you.
-MSGID_BUGS_ADDRESS =
+MSGID_BUGS_ADDRESS = https://translate.sympa.community
```
Comment (S1aa) [inline po/sympa/Makevars:46]: support/xgettext.pldi

**Comment by 080switch** [comment]
Free Trade Fair
For Industry Resource Prelude
@irpoutlook.çom
Screenshot_20240503-095112
Screenshot_20240503-095112

**Comment by 0137248506** [comment]
Screenshot_20240504-211017

**Review by Tt954ggth — COMMENTED** [review]
Эта работа распространяется по лицензии CC BY-NC-ND 4.0

**Pull request merged**